validator/validators/performance/performance_validator.go

```diff
@@ -4,13 +4,32 @@
 package performance
 
 import (
+	"fmt"
+	"log"
+	"strings"
 	"time"
 
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/cenkalti/backoff/v4"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+
 	"github.com/aws/amazon-cloudwatch-agent-test/internal/awsservice"
 	"github.com/aws/amazon-cloudwatch-agent-test/internal/common"
 	"github.com/aws/amazon-cloudwatch-agent-test/validator/models"
 )
 
+const (
+	ServiceName      = "AmazonCloudWatchAgent"
+	DynamoDBDataBase = "CWAPerformanceMetrics"
+)
+
+var (
+	// The default unit for these metrics is byte. However, we want to convert to MB for easier understanding
+	metricsConvertToMB = []string{"mem_total", "procstat_memory_rss", "procstat_memory_swap", "procstat_memory_data", "procstat_memory_vms", "procstat_write_bytes", "procstat_bytes_sent"}
+)
```

Comment:
I agree with you regarding `hashmap` in terms of getting key (time complexity: O(1)) However, small collections of items will have the same (O(n) with small collections will ~ to O(1)) and I would avoid creating `empty-struct` since its only served as a vague placeholder as much as possible. Simplicity in exchange (performance won't be the case here though)